joinfaces-autoconfigure/src/main/java/org/joinfaces/autoconfigure/rewrite/RewriteAutoConfiguration.java

```diff
@@ -69,6 +69,17 @@
 		});
 	}
 
+	/**
+	 * This {@link SpringBootAnnotationConfigProvider} adds a
+	 * {@link org.ocpsoft.rewrite.servlet.config.HttpConfigurationProvider} which scans for Rewrite annotations within
+	 * the classpath.
+	 * @return rewrite annotation scanner
+	 */
+	@Bean
+	public SpringBootAnnotationConfigProvider annotationConfigProvider() {
```

Comment:
Does it really work to declare the ConfigurationProvider as _Spring Bean_?

All other implementations of Rewrite SPI's are declared using files in `META-INF/services/`.

Reply:
Yes, it is enough, that is what the Joinfaces SpringBootServiceLocator is for.

Reply:
I see, but we should be careful with this.
Rewrite will find the implementation class, but won't use the _Spring Bean_ instance. Rewrite will create its own instance of the class.

In this case, this will not be a problem, because no state is injected into the spring bean instance.

Reply:
I've added #624 so Rewrite can use the actual spring beans.